blocks/block-edit/index.js

```diff
@@ -34,18 +37,20 @@ export function BlockEdit( props ) {
 
 	// `edit` and `save` are functions or components describing the markup
 	// with which a block is displayed. If `blockType` is valid, assign
-	// them preferencially as the render value for the block.
+	// them preferentially as the render value for the block.
 	const Edit = blockType.edit || blockType.save;
 
 	// For backwards compatibility concerns adds a focus and setFocus prop
 	// These should be removed after some time (maybe when merging to Core)
 	return (
-		<Edit
-			{ ...props }
-			className={ className }
-			focus={ props.isSelected ? {} : false }
-			setFocus={ noop }
-		/>
+		<AutocompletionContext.Provider value={ { autocompleters } }>
```

Comment:
Similar note to above, we're passing a new object reference every single render, which will break any attempt at purity reconciliation if it exists (and based on my limited understanding of React context providers, likely trigger a re-render to all consumers). Can we not have this prop shaped as an object? Simpler type for props is better.

Reply:
I'll test this. My believe was that it would only affect the provider's children. I'm using the provider incorrectly if this affects anything other than descendant consumers.